aws/resource_aws_msk_cluster.go

```diff
@@ -206,6 +205,54 @@ func resourceAwsMskCluster() *schema.Resource {
 				Type:     schema.TypeInt,
 				Required: true,
 			},
+			"open_monitoring": {
+				Type:             schema.TypeList,
+				Optional:         true,
+				DiffSuppressFunc: suppressMissingOptionalConfigurationBlock,
+				MaxItems:         1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"prometheus": {
+							Type:             schema.TypeList,
+							Optional:         true,
```

Comment:
Is it intended to make `prometheus` optional here? In AWS schema, this is a required element https://docs.aws.amazon.com/msk/1.0/apireference/clusters.html#clusters-model-openmonitoringinfo

Reply:
Fixed, thanks!

Reply:
Fixed, thanks!